reddit_experiments/__init__.py

```diff
@@ -301,6 +301,12 @@ def expose(
         :param kwargs: Additional arguments that will be passed to logger.
 
         """
+        if variant_name is None or variant_name == "":
```

Comment:
applied this retroactively for Experiments SDK, in case devs update package version but still have calls to old `Experiments.expose()` on their system that may be misconfigured